islandora_audio.module

```diff
@@ -236,3 +236,35 @@ function islandora_audio_file_mimetype_mapping_alter(&$mapping) {
   unset($mapping['extensions']['mp3']);
   $mapping['extensions']['mp3'] = $code;
 }
+
+/**
+ * Implements hook_help().
+ */
+function islandora_audio_help($route) {
+  switch ($route) {
+    // Main help page for the module.
+    case 'help.page.islandora_audio':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Islandora Audio module is one of the many Islandora Solution Packs. This module allows for the creation of audio collections which can be populated by uploading WAV and MP3 files as objects within Islandora. An audio file can then be made available for download by users when viewing the corresponding object. See the <a href=":wiki">Audio Solution Pack wiki*</a> for more information.', [
```

Comment:
We used a different convention in the Document SP https://github.com/discoverygarden/islandora_document/pull/7/files#diff-f01edc5bbc1e084b295092ea3d308404R181 for collections. We should use one; maybe how it is in audio is better.

Reply:
Are you talking about the specific wording "This module allows for the creation of audio collections..."? Or the spacing in t()? or the general wording of the whole thing? ...all of the above?

Reply:
Discussed referring to collections more generally, as content models can be included in any collection, and I will modify other help pages to follow this convention.